content/en/synthetics/guide/email-validation.md

```diff
@@ -13,59 +13,56 @@ further_reading:
-
-Datadog [Browser Tests][1] allow you to:
-
-- Confirm an email was correctly sent following up on an in-app trigger
-- Check an email's content
-- Click on the links located within the sent emails in order to navigate to other URLs and validate your whole flows, including web and email steps
-
-To perform email validation inside a Datadog Browser test:
+Maintaining a great user experience on your website includes ensuring that your application's email mechanisms are working properly. 
 
 ## Create an email variable
 
-After filling your Browser test details (start URL, devices, locations, frequency, notification), navigate to **Save Details & Record Test**, select **Variables** and then select **Email** from the email variable dropdown list:
+To add an email variable called `EMAIL`:
+
+1. Click **Variables** and select **Email** from the dropdown menu. 
+2. Click **Add Variable** to make the variable available for you to use when you start recording. 
 
 {{< img src="synthetics/guide/email-validation/adding-variable.mp4" alt="Create an email variable" video="true"  width="100%">}}
 
-In the above example, an email variable called `EMAIL` is created. The newly created email variable generates a unique mailbox maintained by Datadog at every test execution. This is key to avoid any conflicts among your Browser test runs.
+The email variable generates a unique mailbox maintained by Datadog at every test execution, which enables your browser tests to run without conflicts.
 
 ## Record steps
 
-In the upper left corner of the UI, click the **Start Recording** button and record the steps leading to the email being triggered using your freshly created email variable. Use the hand icon to input variables in the text inputs of the form.
+Once you have created an email variable, you can [confirm an email was sent correctly](#confirm-the-email-was-sent) after an in-app trigger, check an email's content, and click on links located in sent emails to [navigate to other URLs](#navigate-through-links-in-an-email) and validate entire user flows including web and email steps. 
+
+Click **Start Recording** and record all of the steps leading up to the email being triggered with your email variable. Click the hand icon in a variable to inject its value into the text input of a form or field.
 
 {{< img src="synthetics/guide/email-validation/record-steps.mp4" alt="Record your steps" video="true"  width="100%">}}
 
-The steps to fill the form are recorded and the click on the `Sign up` button triggers the email. An email is sent to the Datadog mailbox that was created for this recording session as a result (for example, `838-n3q-q2y.6238933596@synthetics.dtdg.co` in this example).
+After recording your steps to complete the form, click the **Sign Up** button to trigger an email notification. An email tailored to this recording session is sent to the Datadog mailbox, for example: `838-n3q-q2y.6238933596@synthetics.dtdg.co`.
```

Comment:
```suggestion
After recording your steps to complete the form, click the **Sign Up** button to trigger an email notification. An email tailored to this recording session is sent to the Datadog mailbox, for example, `838-n3q-q2y.6238933596@synthetics.dtdg.co`.
```

Reply:
I've never seen an example like this. Just want to validate that's something we can use in the docs and doesn't give away something shouldn't be in the docs.